AdyenCard/Form/FormCardNumberContainerItem.swift

```diff
@@ -55,6 +67,12 @@ internal final class FormCardNumberContainerItem: FormItem, AdyenObserver {
     
     internal func update(brands: [CardBrand]) {
         numberItem.update(brands: brands)
+        
+        guard showsSupportedCardLogos else {
```

Comment:
exiting here seems dangerous if we want to add some other logic below that could be for both cases. Maybe if else block for this and the below related logic?